### PR TITLE
Start tests at the top of the minute

### LINKED ISSUE #27738: Test Failure: com.ibm.ws.logging.fat.TimeBasedLogRolloverTest.testTimedRolloverXMLUpdates
RTC 298162

```
testTimedRolloverXMLUpdates:componenttest.exception.TopologyException: 2023-12-09-19:15:44:562 The necessary feature timedexit-1.0 was not enabled. Please include fatTestPorts.xml or fatTestCommon.xml in the server.xml for server com.ibm.ws.logging.timedrolloverxml.
at componenttest.topology.impl.LibertyServer.validateTimedExitEnabled(LibertyServer.java:2878)
at componenttest.topology.impl.LibertyServer.validateServerStarted(LibertyServer.java:2848)
at componenttest.topology.impl.LibertyServer.startServerWithArgs(LibertyServer.java:1885)
at componenttest.topology.impl.LibertyServer.startServerAndValidate(LibertyServer.java:1429)
at componenttest.topology.impl.LibertyServer.startServerAndValidate(LibertyServer.java:1411)
at componenttest.topology.impl.LibertyServer.startServerAndValidate(LibertyServer.java:1398)
at componenttest.topology.impl.LibertyServer.startServer(LibertyServer.java:1145)
at com.ibm.ws.logging.fat.TimeBasedLogRolloverTest.setUp(TimeBasedLogRolloverTest.java:124)
at com.ibm.ws.logging.fat.TimeBasedLogRolloverTest.testTimedRolloverXMLUpdates(TimeBasedLogRolloverTest.java:229)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:203)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:363)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:177)
```

### PR DESCRIPTION
**Pull request by fmhwong**
Fixes #27738 

Make every test to start at the top of the minute so that the server has enough time to start up before the one minute log rollover.

### CODE REVIEW

**Comment by fmhwong** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.logging_2_fat

**Review by Channyboy — DISMISSED** [review]
lgtm

File: dev/com.ibm.ws.logging_2_fat/fat/src/com/ibm/ws/logging/fat/TimeBasedLogRolloverTest.java [hunk 1]
```diff
@@ -1,49 +1,38 @@
...
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.InputStreamReader;
 import java.net.HttpURLConnection;
 import java.net.URL;
-import java.util.List;
+//import com.ibm.ws.logging.internal.impl.DateFormatProvider;
```
Comment (Channyboy) [inline dev/com.ibm.ws.logging_2_fat/fat/src/com/ibm/ws/logging/fat/TimeBasedLogRolloverTest.java]: remove import?

**Comment by fmhwong** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.logging_2_fat

**Comment by fmhwong** [comment]
#libby

**Review by Channyboy — APPROVED** [review]
lgtm

**Pull request merged**